lib/connection-browser.js

```diff
@@ -24,11 +24,14 @@ Connection.prototype._init = function(callback) {
 /**
  * Create new serialport instance for the Arduino board, but do not immediately connect.
  */
-Connection.prototype._setUpSerial = function(callback) {
+Connection.prototype._setUpSerial = function(port, callback) {
```

Comment:
can we take it as optional argument

Reply:
nhaa we dont need this param at all. Missed to remove, will remove,